sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleTracingIT.java

```diff
@@ -39,24 +43,43 @@
 
   @Test
   public void shouldHaveInstrumentation() {
-    Datastore datastore = getDatastore();
+    Datastore datastore = getMorphiaBundle().datastore();
     Person person = new Person().setAge(18).setName("Max");
     datastore.save(person);
 
     MockTracer tracer = getMockTracer();
-    assertThat(tracer.finishedSpans())
-        .extracting(MockSpan::operationName)
-        .contains("createIndexes", "insert");
-    assertThat(
-            tracer.finishedSpans().stream()
-                .map(s -> s.tags().keySet())
-                .flatMap(Set::stream)
-                .noneMatch(Tags.DB_STATEMENT::equals))
-        .isTrue();
+    await()
```

Comment:
Can we add a timeout either to the waiting or to the test itself?

Reply:
What do you mean with timeout? By default, await comes with a timeout of 10 seconds, I don't think that we should specify it in every test class and every usage of await.

Reply:
Good to know. It‘s hard to see that in a review on GitHub without an IDE.